Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html

```diff
@@ -138,8 +138,10 @@
     }
 }];
 
-var viewer = new Cesium.Viewer('cesiumContainer');
-viewer.dataSources.add(Cesium.CzmlDataSource.load(czml));
+Cesium.GroundPrimitive.initializeTerrainHeights().then(function() {
```

Comment:
Why is this needed here?  The Entity visualization system should already be doing this automatically for the user, if it's not then we need to add code to make that happen; not force users to do it themselves.